Dockerfile

```
@@ -120,7 +120,13 @@ RUN \
 	&& apk add --no-cache --virtual .geoip2-build-deps \
 		libmaxminddb-dev \
 	&& apk add --no-cache --virtual .njs-build-deps \
-		readline-dev
+		libedit-dev \
+		libxml2-dev \
+		libxslt-dev \
+		openssl-dev \
+		pcre-dev \
+		readline-dev \
+		zlib-dev
```

```suggestion
		zlib-dev \
	&& git config --global init.defaultBranch master
```

Remove the following warning when running `git init`:

```
2024-11-25T21:03:13.2711083Z hint: Using 'master' as the name for the initial branch. This default branch name
2024-11-25T21:03:13.2712170Z hint: is subject to change. To configure the initial branch name to use in all
2024-11-25T21:03:13.2712860Z hint: of your new repositories, which will suppress this warning, call:
2024-11-25T21:03:13.2713593Z hint:
2024-11-25T21:03:13.2713987Z hint: 	git config --global init.defaultBranch <name>
2024-11-25T21:03:13.2714369Z hint:
2024-11-25T21:03:13.2714796Z hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
2024-11-25T21:03:13.2715501Z hint: 'development'. The just-created branch can be renamed via this command:
2024-11-25T21:03:13.2715991Z hint:
2024-11-25T21:03:13.2716510Z hint: 	git branch -m <name>
```